src/css/custom.css

```
@@ -271,4 +273,11 @@ blockquote p::after,
 .markdown blockquote p::after {
   content: none !important;
   display: none !important;
+}
+
+p{
+  line-height: 1.5;
+}
+.markdown img {
+  margin-bottom: 0; /* Adjust this value as needed */
```

This will be half of the current value, 0 was a bit too close in my opinion.
```suggestion
  margin-bottom: 1rem;
```